src/Symfony/Component/Cache/Marshaller/IdentityMarshaller.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Marshaller;
```

Comment:
don't miss renaming the file+namespace for `Symfony\Component\HttpFoundation\Session\Storage\Handler`

Reply:
you mean we move the `IdentityMarshaller` to HttpFoundation ?

Reply:
yes

Reply:
Done, I also added unit tests